cmake/NSIS.template.in

```diff
@@ -555,6 +555,14 @@ Section /o "Open OBJ files with Cura"
     WriteRegStr HKCR "Cura OBJ model file\shell\open\command" "" '"$INSTDIR\Cura.exe" "%1"'
 SectionEnd
 
+Section /o "Open 3MF files with Cura"
+    WriteRegStr HKCR .3mf "" ""
```

Comment:
Shouldn't we merge this first and see whether it works?
Adding proper descriptions here should go into a new JIRA issue, I think. CURA-1919 is only about adding 3mf file association.